software/acpid.txt

```diff
@@ -0,0 +1,155 @@
+ACPID
+________________________________________________________________________________
+
+Acpid is a daemon that executes certain actions whenever ACPI events are
+recieved. Depending on your hardware and kernel configuration, these events
+include closing a laptop lid, connecting to an AC power adapter, pressing
+certain buttons and more.
+
+
+Index
+________________________________________________________________________________
+
+- Usage                                                                    [0.0]
+    - Kernel Configuration                                                 [0.1]
+
+- Busybox acpid                                                            [1.0]
+
+- Acpid2                                                                   [2.0]
+    - Finding Events                                                       [2.1]
+    - Handling Events                                                      [2.2]
+
+
+[0.0] Usage
+________________________________________________________________________________
+
+KISS Linux offers two options for acpid management: busybox acpid, which is
+installed by default, and acpid2, which can be installed from the acpid
+package. For both acpid implimentations, you will need to enable a few
+ACPI-related kernel options and optionally the runit service to manage it.
+
+To enable the acpid runit service run:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ ln -s /etc/sv/acpid/ /var/service/                                       |
+|   $ sv up acpid                                                              |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+Whenever changes are made, restart acpid for them to take effect.
+
++--------------------------------------------------------------------------+
+|                                                                          |
+|   $ sv restart acpid                                                     |
+|                                                                          |
++--------------------------------------------------------------------------+
+
+
+    [0.1] Kernel Configuration
+    ____________________________________________________________________________
+
+    For now, check the Gentoo Wiki for details. [0] The important information
+    will be added to this page soon.
+
+
+[1.0] Busybox acpid
+________________________________________________________________________________
+
+Busybox provides a very limited acpid program, that only works well for laptop
+lid events. Everytime the lid is closed, acpid will execute /etc/acpid/LID/00000080.
```

Comment:
Line exceeds 80 columns.

Reply:
Didn't you say above to leave rewrap this paragraph with longer lines?

Reply:
I meant that there was space at the end of each line (within the 80 column limit) where the paragraph could be re-wrapped (making each line longer while within limits).